changes/bug31343

```diff
@@ -0,0 +1,9 @@
+  o Minor bugfixes (compilation):
+    - Avoid using labs() on time_t, which can cause compilation warnings
+      on 64-bit Windows builds.  Fixes bug 31343; bugfix on 0.2.4.4-alpha.
+
+  o Minor bugfixes (clock skew detection):
+    - Don't believe clock skew results from NETINFO cells that appear to
+      arrive before the VERSIONS cells they are responding to were sent.
+      Previously, we would accept them up to 3 minutes "in the past".
```

Comment:
Previously, we would calculate the skew up to 3 minutes in the past, but never warn on it, because we only warned on skews over 1 hour.

Reply:
That's not quite right either -- there are two times here.  One is the skew, and one is the allowable delay in the netinfo cell.  It's possible to see a 1 hour skew even if the netinfo cell delay is negative.